app/controllers/candidate_interface/submitted_application_form_controller.rb

```diff
@@ -21,5 +21,9 @@ def carry_over
       CarryOverApplication.new(current_application).call
       redirect_to candidate_interface_continuous_applications_details_path
     end
+
+    def redirect_to_application_form_unless_submitted
+      redirect_to candidate_interface_start_carry_over_path unless current_application.submitted?
+    end
```

Comment:
I will left this as it is, and do a change into "organise our redirects" trello card because I think this redirect shouldn't exist the moment we have everything under one page and organised.